docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -24,20 +24,54 @@
 .. _vivarium_best_practices_vivarium_and_other_models:
 
 =========================================================
-Vivarium Versus Other Types of Models
+What is Vivarium Good For?
 =========================================================
 
 .. contents::
    :local:
    :depth: 1
 
+What are individual-based microsimulations?
+-------------------------------------------
+
+Vivarium is the Simulation Science team's platform for individual-based microsimulation (also known as agent-based simulation), which is a technique for leveraging information about attributes of individuals in a population (such as demographics, risk factors, disease occurrence, and intervention coverage) and relationships between these attributes to estimate health outcomes in a synthetic environment over time.
```

Comment:
nit: it's generally nice to wrap long lines like this into multiple. Makes it a little easier to deal with in git: if someone edits it later, their diff will only be on the lines they touched.

Reply:
Also, maybe for a future PR, but this section uses a lot of big words; it'd be great to simplify and/or add pictures to explain this concept to a microsim newcomer. I think we have PowerPoint slides about this, which might be able to be repurposed here.